open_dread_rando/game_patches.py

```diff
@@ -51,3 +53,31 @@ def apply_game_patches(editor: PatcherEditor, configuration: dict):
             },
             "mapProps"
         )
+    
+    if configuration["linear_damage_runs"] and configuration["linear_dps"] > 0:
+    
+        from math import ceil   #couldnt find a better way to do this without loops
+        
+        #This section should find a tick speed between 0.2 and 0.4 and determine a damage amount that should calculate to be the total DPS
+        #This *could* be customizable I suppose, but would require more UI work and the game is very picky on what values it allows. Only whole number damage values, and the tick speed cannot be too low or else it caps.
+        
+        damage = ceil(float(configuration["linear_dps"])/5)
```

Comment:
a comment explaining the division by 5 would be useful

Reply:
The comment on line 61 is technically the explanation, since the division by 5 results in a whole number damage tick which is used to calculate the minimum possible value for tick rate between 0.2 and 0.4, but I could make that more clear for sure